src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -75,6 +75,7 @@ static constexpr std::string_view MoveFocusLeftKey{ "moveFocusLeft" }; // Legacy
 static constexpr std::string_view MoveFocusRightKey{ "moveFocusRight" }; // Legacy
 static constexpr std::string_view MoveFocusUpKey{ "moveFocusUp" }; // Legacy
 static constexpr std::string_view MoveFocusDownKey{ "moveFocusDown" }; // Legacy
+static constexpr std::string_view OpenSearchBoxKey{ "openSearchBox" };

```

Comment:
I think I'd prefer if this was bound to `find`, or maybe `toggleFind`, especially if the default keybinding is <kbd>Ctrl+F</kbd>

Reply:
Changed to ToggleFind